module/VuFind/src/VuFind/I18n/Translator/Loader/ExtendedIni.php

```diff
@@ -37,218 +38,209 @@
  * @category VuFind
  * @package  Translator
  * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Sebastian Kehr <kehr@ub.uni-leipzig.de>
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org Main Site
  */
-class ExtendedIni implements FileLoaderInterface
+class ExtendedIni implements RemoteLoaderInterface
 {
+    const EXTEND = '@extend';
+
+    const FALLBACK = '@fallback';
```

Comment:
I suggest dropping this directive and instead allow to configure inheritance between languages.

Reply:
Agreed, as noted above.